src/models/network/NetworkBaseController.js

```diff
@@ -83,7 +84,8 @@ export default class NetworkBaseController {
   }
 
   async uploadContract(contractAlias, contractName) {
-    const contractClass = Contracts.getFromLocal(contractName);
+    console.log(contractBuildDirectory())
+    const contractClass = Contracts._getFromBuildDir(contractBuildDirectory(), contractName);
```

Comment:
i understand we shouldn't be calling `_getFromBuildDir` as that's meant to be a private function, but perhaps this can lead to a proposed change in `zos-lib` as well to make that `getFromBuildDir` public function